### PR TITLE
TLE processing improvements

### PR DESCRIPTION
**Pull request by tomaszmrugalski**
This PR introduces some code changes related to TLE processing. The long term goal is to be able to georeference received images. But it's complex target and we won't get there in one step. So far the code changes cover:

- TLE information is displayed in observations list and in observation details
- Several cosmetic changes related to how AOS, TCA and LOS info is presented
- Test data extended with observation 1276
- Fixed bugs in create_charts script
- ~~implemented noaa_process script:~~ (moved to separate repo)

### CODE REVIEW

**Comment by tomaszmrugalski** [comment]
Hey @fivitti, the code is now ready for review. Some of the marks have tooltips. Check them out.

I've moved the script to a separate repo.

File: server/templates/obslist.html [hunk 1]
```diff
@@ -64,20 +64,34 @@ <h3>Observations</h3>
...
+            <td>{{ obs.los.isoformat(" ", "seconds") }}</td>
             <td>
                 <a href="https://www.n2yo.com/satellite/?s={{ obs.sat_id }}">
                     {{ obs.sat_name }}
                 </a>
             </td>
+            <td>
+                {% if obs.tle %}
+                <span class="tle glyphicon text-success glyphicon-ok" data-toggle="tooltip" data-html="true" boundary="window"
+                      title="<b>TLE</b><br/>{{ obs.tle[0]}}<br/>{{ obs.tle[1] }}"></span>
```
Comment (fivitti) [inline server/templates/obslist.html]: Why you place the TLE in tooltip? It takes up a lot of space and isn't human-readable. When TLE is in tooltip user cannot copy it too. I think that icon with TLE status is enough on observation list.

File: server/templates/stations.html [hunk 2]
```diff
@@ -18,8 +18,8 @@ <h3>Ground stations</h3>
             <td><a href="/station/{{ s.station_id }}">{{ s.name }}</a></td>
             <td>{{ s.coords }}</td>
             <td>{{ s.cnt }} [<a href="/obslist?station_id={{s.station_id}}">list</a>]</td>
-            <td>{{ s.registered }}</td>
-            <td>{{ s.lastobs }}</td>
+            <td>{{ s.registered.isoformat(" ", "seconds") }}</td>
```
Comment (fivitti) [inline server/templates/stations.html:21]: You put the same datetime format in several places. Maybe you want to use [template filter](https://stackoverflow.com/a/4830620) for define format in one place. It isn't important for me and you may to ignore this note. It is here only if you want to learn something new.

**Review by fivitti — APPROVED** [review]
I don't found any bugs. Code is good. Thanks for your time!
I add only few tips for better UI (in my opinion) and improve the code.
When you fill them or decide to ignore then you may immediately merge this branch to master.

File: server/templates/obslist.html (hunk 1 above)
Comment (tomaszmrugalski) [reply server/templates/obslist.html]: Ok, removed.

File: server/templates/stations.html (hunk 2 above)
Comment (tomaszmrugalski) [reply server/templates/stations.html:21]: Cool trick. Added template filter.

**Pull request merged**